keg_login/middleware.py

```diff
@@ -0,0 +1,27 @@
+import flask
+
+
+class LockedOutMiddleware(object):
+    lock_endpoint = 'keg_login.lock'
+
+    session_key_name = 'keg-login.lockout'
+
+    default_excluded_routes = [
+        'static',
+        'keg_login.logout',
+        lock_endpoint,
+    ]
+
+    def __init__(self, exclude=[]):
+        self.exclude_routes = exclude + self.default_excluded_routes
```

Comment:
Yea, that wasn't intentional. I just missed it while I was coding. Thanks @bladams.